bootstrap

```diff
@@ -46,6 +46,7 @@ Configuration:
     --disable-stats                 disables internal TileDB statistics
     --disable-avx2                  disables use of AVX2 instructions
     --disable-webp                  disables building of webp library and linkage test
+    --disable-vcpkg                 disables use of vcpkg for downloading and building dependencies
```

Comment:
Should probably remove the `--enable-vcpkg` option as well?

Reply:
Oh, or warn that its a no-op like you've done in the Windows bootstrap.ps1

Reply:
Fixed.